app/services/recommendation_service.py

```
@@ -109,18 +446,282 @@ async def _get_product_details(self, product_ids: List[int]) -> Dict[int, Dict[s
+                    "왁스/젤/무스/토닉", "스프레이/픽서", "헤어퍼퓸"
+                ],
+                "바디케어": [
+                    "바디케어", "바디워시", "바디스크럽", "입욕제", "샤워/입욕", "비누",
+                    "로션/오일/미스트", "바디미스트", "바디오일", "핸드케어", "핸드크림", "핸드워시",
+                    "풋케어", "풋크림", "풋샴푸", "발냄새제거제", "발각질제거제", "발관리용품",
+                    "제모/왁싱", "면도기/면도날", "제모크림", "스트립/왁스", "제모기기", "남성 쉐이빙",
+                    "데오드란트", "데오스틱", "데오롤온", "데오스프레이", "쿨링/데오시트", "베이비"
+                ]
+            }
+            
+            logger.info(f"🏗️ 카테고리 계층 매핑 생성 완료")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)